src/pages/Stores/components/BlockStore/BlockStore.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './styles.scss';
+
+import Table from '@/components/Table/Table';
+import { Store } from '@/context/WorkspaceContext/types';
+import { useWorkspace } from '@/hooks/useWorkspace';
+
+const BlockStore = () => {
+  const { activeWorkspace } = useWorkspace();
+
+  const renderBlocks = (store: Store) => {
```

Comment:
Can we put a guard statement at the top here.
As if there are no block stores in the workspace, the whole page will crash as this code maps over an object, that if `null` would crash the site